src/test/unittest/unittest.sh

```diff
@@ -247,9 +266,10 @@ export CHECK_POOL_LOG_FILE=check_pool_${BUILD}_${UNITTEST_NUM}.log
 # example, to list all *.log files in the current directory
 #	get_files ".*\.log"
 function get_files() {
+	store_set_e
 	set +e
```

Comment:
I think it would more useful to have a wrapper around store_set_e;set+e - something like "disable_fail_on_error". And then all uses of restore_set_e could become "restore_fail_on_error".